README.md

```diff
@@ -1,4 +1,4 @@
-# Django support for Sublime Text 2/3
+# Django support for Sublime Text 4
```

Comment:
`.sublime-syntax` was added in ST3: https://docs.sublimetext.io/guide/extensibility/syntaxdefs.html  I've also tried this and it does work with version 3.